img.py

```diff
@@ -57,8 +70,34 @@ def import_images(self):
 			self.origIndex = 0
 		self.update_curr_img_obj()
 
-	def setup_GUI(self):
+	def import_images_alt(self):
+		# *******************************************************
+		#
+		# uncomment this line to clear the list of images on open
+		# 
+		# *******************************************************
+		# self.origList = []

```

Comment:
NITPICKY: Maybe have a function that we can call through a key stroke that clears the list for us.